port.yaml

```diff
@@ -0,0 +1,36 @@
+---
+# (required) Lifecycle values can be one of pre-alpha, alpha, beta, release-candidate, generally-available or stable
```

Comment:
The list says `generally-available` but the value is `generally_available`. We should get that in sync. 😉

Reply:
@Niceplace, I think we discussed this one during our previous sync, the one before offsite. We agreed to skip lifecycle, right?